src/sentry/integrations/slack/webhooks/base.py

```diff
@@ -19,10 +19,17 @@
 NOT_LINKED_MESSAGE = "You do not have a linked identity to unlink."
 ALREADY_LINKED_MESSAGE = "You are already linked as `{username}`."
 
+from sentry.utils import metrics
+
+from ..utils import logger
```

Comment:
should we do `logger = logging.getLogger(__name__)`

Reply:
when should we use the common logger + add additional namespace when logging vs make a new one?